lisa/tools/lsvmbus.py

```diff
@@ -87,6 +87,8 @@ def parse(self, raw_str: str) -> Any:
 
 
 class Lsvmbus(Tool):
+    __pattern_not_found = re.compile(r"(.*not found.*)")
```

Comment:
let it be more completed to avoid ignoring unknown issue. below one is better.

`WARNING: lsvmbus not found for kernel`